routes/web.php

```diff
@@ -79,6 +80,12 @@
     Route::get('licenses/{id}', [LicenseController::class, 'getLicensebyId'])
                 ->name('license');
 
+    // Authors
+    Route::get('authors', [AuthorController::class, 'getAuthorsbyDOI'])
```

Comment:
This method should be retrieving the authors associated with project/study/dataset